_raml/apps/products/products.raml

```diff
@@ -34,6 +34,18 @@ get:
         application/json:
           example: !include examples/get_product_response.json
           schema: !include schema/get_product.schema.json
+post:
+  description: Creates a new product.
+  securedBy: [products_write]
+  is: [ locale, currency ]
+  body:
```

Comment:
As already mentioned, we have to wait until @timosen clarified with Domingo to implement a request attribute table.